apollo-normalized-cache-sqlite/src/commonTest/kotlin/com/apollographql/apollo/cache/normalized/sql/createDriver.kt

```diff
@@ -0,0 +1,5 @@
+package com.apollographql.apollo.cache.normalized.sql
+
+import com.squareup.sqldelight.db.SqlDriver
+
+expect fun createDriver(): SqlDriver
```

Comment:
By having this in `commonTest`, I was able to keep all tests in common module and run them against multiple platforms.